src/printapp/static/css/style.css

```diff
@@ -1,3 +1,4 @@
+/* Note: 1em = 16px  */
```

Comment:
We don't need this comment. It's more about how css works than how our app works.